performant-software/Annotation-Studio
# Preserve line breaks in uploaded/converted document

**What this PR does:**

- Update styling to preserve line breaks in text when viewing or editing documents (including uploaded and converted documents)
- Add precompiled assets


**Steps to test:**

1. Upload a new document from an OCR-enabled pdf file
2. Open newly created document (may require a page refresh); observe that line breaks in the text of the newly created document are preserved

**Visual change(s):**

Before:

<img width="1023" alt="Screen Shot 2021-03-06 at 1 52 11 AM" src="https://user-images.githubusercontent.com/20568337/110199626-de61c480-7e1e-11eb-9f0e-e4967eac8d24.png">

After:

<img width="1023" alt="Screen Shot 2021-03-06 at 1 52 21 AM" src="https://user-images.githubusercontent.com/20568337/110199631-e457a580-7e1e-11eb-91b0-accc6e5ae220.png">
